tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -37,15 +41,19 @@
 
     from transformers import (
         Mamba2ForCausalLM,
+        Mamba2ForSequenceClassification,
```

Comment:
Are you sure the imports are correct? Doesn't seem to work on my end; possibly due to the false name, some forgotten __init__.

Reply:
Should be resolved when applying the latest comment of mine in mamba2.init as a "," was missing.